perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/pwdmgr/GenericPasswordManagerModule.java

```diff
@@ -160,9 +162,9 @@ public void checkLoginFormat(PerunSession sess, String login) throws InvalidLogi
 	@Override
 	public void checkPasswordStrength(PerunSession sess, String login, String password) throws PasswordStrengthException {
 
-		if (StringUtils.isBlank(password)) {
-			log.warn("Password for {}:{} cannot be empty.", actualLoginNamespace, login);
-			throw new PasswordStrengthException("Password for " + actualLoginNamespace + ":" + login + " cannot be empty.");
+		if (StringUtils.length(password) < MINIMUM_PASSWORD_LENGTH) {
+			log.warn("Password for {}:{} must be at least 8 characters long.", actualLoginNamespace, login);
```

Comment:
Why not use the MINIMUM_PASSWORD_LENGTH in the log functions as well?

Reply:
Nice suggestion, thank you!